components/Footer/index.tsx

```diff
@@ -45,10 +45,12 @@ const Footer = styled(({ className }) => {
     );
 })`
     display: flex;
+    align-items: center;
     position: fixed;
+    height: 5vh;
```

Comment:
can we set a max height on these. Since when changing it to vh on longer screens it can get quite large